packages/observable-dom/src/JSDOMRunner.ts

```diff
@@ -26,6 +26,36 @@ class RejectionResourceLoader extends ResourceLoader {
   }
 }
 
+export type ResourceURL = string | RegExp;
+
+// This allows JSDOM to load resources if their URLs are specified in the urls array.
+class AllowListResourceLoader extends ResourceLoader {
+  private urls: ResourceURL[]
+
+  constructor(urls: ResourceURL[], opts?: ResourceLoaderConstructorOptions) {
+    super(opts)
+    this.urls = urls
+  }
+  public fetch(url: string, opts?: FetchOptions): AbortablePromise<Buffer> | null {
+    const allow = this.urls.every(allowedURL => {
```

Comment:
```suggestion
    const allow = this.urls.some(allowedURL => {
```

I think this is meant to be "if any of the rules / addresses allow this resource", right?

Reply:
Ah yes, thanks.

That's on me only testing with one resource URL :man_facepalming:.